tasks/darwin-install.yml

```diff
@@ -72,29 +71,6 @@
   changed_when: killadbresults.rc == 0
   failed_when: killadbresults.rc != 0 and killadbresults.rc != 1
   become: yes
-  
-- name: Get running jmx processes
-  shell: "ps -ef | grep -v grep | grep {{ jmx_port }} | awk '{print $2}'"
-  register: running_processes
-  become: yes
-
-- name: Kill running jmx processes
```

Comment:
Please resolve the conflicts, there is changes pushed to develop branch.
please get pull the latest changes from develop branch and raise the PR

Reply:
Resolved the conflict by accepting relevant changes from develop.